data/plugins.json

```diff
@@ -238,5 +237,14 @@
     "ios": "10",
     "opera": "45",
     "electron": "1.7"
-  }
+  },
+  "transform-async-generator-functions": {},
```

Comment:
These will get filled out once we land https://github.com/babel/babel-preset-env/pull/402